v2/samples/network/v1api20220701/v1api20240101_webapplicationfirewallpolicy.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: network.azure.com/v1api20240101
+kind: WebApplicationFirewallPolicy
```

Comment:
This is a weird place for this resource sample, my plan is to create new versions of all of the networking resources and then I'll move this sample to a 2024 folder

Reply:
Will do that in a separate PR though